dataactvalidator/config/sqlrules/a32_appropriations.sql

```diff
@@ -9,10 +9,9 @@ SELECT approp.row_number,
 FROM appropriation AS approp
 WHERE approp.submission_id = {0}
 	AND EXISTS (
-		SELECT other.row_number,
-			other.tas
+		SELECT 1
 		FROM appropriation AS other
 		WHERE other.row_number <> approp.row_number
-			AND other.tas IS NOT DISTINCT FROM approp.tas
+			AND other.tas_id IS NOT DISTINCT FROM approp.tas_id
```

Comment:
What about changing this line to `AND other.tas_id = approp.tas_id`? I think this is safe to do since previous rules are already generating errors for rows with a null `tas_id`. If I'm reading this correctly, keeping the `IS DISTINCT FROM` has potential to generate a lot of noise if there are multiple rows in a file with NULL `tas_id`.

Historically, there's been a lot of back and forth about the question of: do we show all error messages for all rules, even if multiple rule failures stem from the same data issue. Regardless of the differing opinions on that question, I don't see anything to be gained by potentially allowing false positives.